content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -390,47 +358,4 @@ If you are running vSphere < 6.7U3, the in-tree vSphere cloud provider is recomm
 {{% /tab %}}
 {{< /tabs >}}
 
-For in-depth documentation on the vSphere cloud provider, visit the [vSphere cloud provider docs site](https://cloud-provider-vsphere.sigs.k8s.io).
-
-## IBM Cloud Kubernetes Service
```

Comment:
@bradtopol 👋  This content meets the criteria for removal. Would you like to re-home it? Please let me know within 48 hours.

Reply:
@bradtopol , Did you get a chance to review this change?